package.json

```diff
@@ -4,10 +4,11 @@
   "description": "A template for creating reusable module for Angular 2 and Ionic 2",
   "main": "./dist/index.js",
   "typings": "./dist/index.d.ts",
+  "files": "dist",
   "scripts": {
     "ngc": "ngc",
     "build": "rm -rf aot dist && npm run ngc",
-    "publishPackage": "npm run build && npm publish"
+    "prepublish": "npm run build"
```

Comment:
This causes commands like `npm install` or `yarn add <module>` to start the build process, which is a minor annoyance. Yarn is mostly the annoying part here, since it starts the build process every time you add a module.